### PR TITLE
[doc] typos fix on doc

### PR DESCRIPTION
**Pull request by smeng9**
Add dynamic resource creation/update/delete example to Admin.md
Update some menu items in documentation to indicate some props are depreacted.

### CODE REVIEW

File: docs/Admin.md [hunk 1]
```diff
@@ -507,4 +507,66 @@ function AsyncResources() {
...
+            .then((resources) => resources.forEach((r) => addResource(r)));
+    }, []);
+
+    return (
+        <AdminUI>
+            {Object.keys(resources).map((resource) => (
+                <Resource
+                    name={resources[resource].name}
+                    key={resources[resource].name}
+                    list={schemaToListView(resources[resource])}
```
Comment (fzaninotto) [inline docs/Admin.md:556]: This isn't defined

File: docs/Admin.md [hunk 2]
```diff
@@ -507,4 +507,66 @@ function AsyncResources() {
...
+                />
+            ))}
+            <Resource
```
Comment (fzaninotto) [inline docs/Admin.md:559]: Adding a schemas Resource is overkill in this example IMHO

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: docs/Admin.md [hunk 3]
```diff
@@ -507,4 +507,59 @@ function AsyncResources() {
 }
 ```
 
+### Dynamically create, update, and delete `<Resource>`
```
Comment (fzaninotto) [inline docs/Admin.md:510]: Now that I think about it, I don't think this addition is relevant. We already explain how to add Resources dynamically based on an HTTP request earlier in this chapter. Your addition only explains another way to do it, which is only useful if people want to recreate ra-no-code. And we don't want to explain too many ways to do things, to encourage reimplementing existing modules, or to solve every possible userland requirement in the documentation.

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: docs/Admin.md (hunk 3 above)
Comment (smeng9) [reply docs/Admin.md]: Ok. I can remove this part for now.

Are we going to have the `ra-no-code` documentation mentioned somewhere else?

Comment (fzaninotto) [reply docs/Admin.md]: When we feel that it's ready, yes. For now, it's not yet ready.

**Review by fzaninotto — APPROVED** [review]

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**